packages/services/f-braze-adapter/package.json

```
@@ -35,23 +36,25 @@
     "extends @justeat/browserslist-config-fozzie"
   ],
   "dependencies": {
-    "appboy-web-sdk": "2.5.2",
     "date-fns": "2.16.1",
     "lodash.findindex": "4.6.0",
     "lodash.orderby": "4.6.0",
     "lodash.startswith": "4.2.1",
     "lodash.uniq": "4.5.0"
   },
   "devDependencies": {
+    "@braze/web-sdk": "3.3.0",
     "@vue/cli-plugin-babel": "4.4.6",
     "@vue/cli-plugin-eslint": "3.9.2",
     "@vue/cli-plugin-unit-jest": "4.4.6",
     "core-js": "3.6.5",
     "jest-extended": "0.11.5",
     "mockdate": "3.0.2",
-    "js-cookie": "2.2.1"
+    "js-cookie": "2.2.1",
+    "vite": "2.4.2"
   },
   "peerDependencies": {
-    "js-cookie": ">=2.2.1"
+    "js-cookie": ">=2.2.1",
+    "@braze/web-sdk": "3.3.0"
```

This should be a slightly more accepting version specifier, to allow consumers to use a later version if necessary
```suggestion
    "@braze/web-sdk": "^3.3.0"
```